lld/test/ELF/Inputs/aarch64-reloc-implicit-addend-targets.s

```diff
@@ -0,0 +1,17 @@
+.globl abs16
```

Comment:
In recent tests the convention has been to put all the inputs in one file and use `split-file` to create the test inputs dynamically.

I think that this can be done here.

Reply:
Nice! I hadn't thought of that, and I agree it improves the test significantly, because now I can interleave the expected output with the instructions that generate it.